runtime/dancebox/src/lib.rs

```diff
@@ -1057,6 +1057,8 @@ pub enum ProxyType {
     Registrar = 6,
     /// Allow extrinsics related to Registrar that needs to be called through Sudo
     SudoRegistrar = 7,
+    /// Allow extrinsics from the Session pallet for key management.
+    SessionKeyManagement = 8,
```

Comment:
We don't need this in flashbox?

Reply:
I don't think so. 
This came up as a requirement for node operators to decouple staking from key management, and there will be no external node operators on flashbox.

Reply:
Yeah exactly we could put it but no-one is going to use it